contracts/math/BondingCurve.sol

```diff
@@ -8,6 +8,7 @@ contract BondingCurve {
 
     // Approximate .001x^2+.000 000 000 000 000 000 000 000 0000999x^{8}
     function curve(uint256 n) public view returns (uint256) {
+        require(n > 0, "BondingCurve: starting position cannot be zero");
```

Comment:
Could we return a start value when N == 0?
or have the bonding curve have a constant

Then we don't need to do the +1 on getPrice

Reply:
trying to keep the concerns separate and let the bonding curve be a pure function